org.tlauncher.TLauncher.yaml

```diff
@@ -0,0 +1,101 @@
+---
+app-id: org.tlauncher.TLauncher
+command: tlauncher
+
+base: io.atom.electron.BaseApp
+base-version: "19.08"
+
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  # Launcher/game files
+  - --persist=.tlauncher
+  - --persist=.minecraft
+  # X11 + XShm
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  # OpenGL
+  - --device=dri
+  # Sound/music
+  - --socket=pulseaudio
+  # Download game files/multiplayer
+  - --share=network
+  # Java
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+  - --env=JAVA=/app/jre/bin/java
+
+build-options:
+  append-path: /usr/lib/sdk/openjdk11/bin
+
+modules:
+  # Needed by Minecraft 1.8.2 and up
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.1.tar.gz
+        sha256: 7b99edb7970a1365eaf5bcaf552144e4dfc3ccf510c4abc08569849929fb366e
+
+  - name: gamemode
```

Comment:
I'm not sure but isn't it part of the runtime?

Reply:
Not that I know of, Steam bundles it. Would be a good idea to include it now that we have a portal, though.